### ISSUE TITLE
about weights of model_vgg

### ISSUE THREAD

**Issue by ghost** [imatge-upc/detection-2016-nipsws#19]
Thank you for your code. I have two questions.
1. Is the vgg16_weights.h5, which downloaded from the link provided by you, pretrained  on ImageNet?
2. It seems that the model_vgg is trained at every epoch, but do not be saved. At test time, the weights of model_vgg is still the Initial weights(i.e. vgg16_weights.h5). Why not save it and use it at test time?

**Comment by mani5h** [comment]
Yes, pretrained CNN is used here,using pretrained cnn (vgg16 here) has 2 benefits:First,
learning the Q function is faster because we need to up-date the parameters of the Q-Network only, while using the deep CNN just as a feed-forward feature extractor. Second, the hierarchy of features is trained with a larger dataset,leveraging generic discriminative features in this method.

**Comment by zoupei123456** [comment]
Have you made any changes to the data set voc2012?Why does this error occur when I run the program?
xml.etree.ElementTree.parseError:not well-formed(invalid token):line1,column 0。Could you tell me how to solve this problem?

**Comment by XiaoYigwr** [comment]
Hi，there is a problem that makes me very confused, in the DQN model of the paper, Why did you use 1024*1024 full connection, instead of 4096*4096,or other dimensions. Does 1024*1024 have any special meaning?

**Comment by zhongkedazhouman** [comment]
thank you! i have a question. It seems that the model_vgg is trained at every epoch, but do not be saved. At test time, the weights of model_vgg is still the Initial weights(i.e. vgg16_weights.h5). Why not save it and use it at test time?
